python/neuroglancer/pipeline/task_execution.py

```diff
@@ -9,7 +9,7 @@
 
 
 @click.command()
-@click.option('--tag', default='',  help='kind of task to execute')
+@click.option('--tag', default=None,  help='kind of task to execute')
```

Comment:
The problem with default=None is that click things there is no default value, so it will complain if you don't specify a value